src/main/java/seedu/duke/data/notebook/TagManager.java

```diff
@@ -11,7 +11,10 @@
  */
 public class TagManager {
 
-    private Map<Tag, ArrayList<Note>> tagMap;
+    private static final String STRING_TAG_EMPTY = "There are no tags!";
+    private static final String STRING_TAG_LIST = "Here are the available tags:" + InterfaceManager.LS;
+
+    private final Map<Tag, ArrayList<Note>> tagMap;
```

Comment:
Should it be TAGMAP?
```suggestion
    private final Map<Tag, ArrayList<Note>> TAGMAP;
```